deploy/openshift/deploy-kiali-to-openshift.sh

```diff
@@ -246,7 +246,7 @@ PROTOCOL="$(if [[ $(oc get routes -n ${NAMESPACE} kiali -o jsonpath=\"{.spec.tls
 # Deploy Kiali MonitoringDashboards to OpenShift
 # Note for undeploy script: dashboards are implicitly undeployed when the related CRD is removed
 echo "Deploying Kiali dashboards to OpenShift project ${NAMESPACE}"
-for dashboard in nodejs thorntail vertx-client vertx-eventbus vertx-pool vertx-server
+for dashboard in nodejs thorntail vertx-client vertx-eventbus vertx-pool vertx-server springboot-jvm springboot-tomcat
```

Comment:
remember, none of these exist in the v0.XX branches, so everyone now installing Kiali with these scripts are going to get a WARNING at the end. We should fix this to make this more fault tolerant - if it can't install any of these, it should probably spit out a more gentler message than have a WARNING in all-caps at the end. Just want to avoid users from  getting nervous when seeing "WARNING" spit out during installs. I don't know a good approach, but I just have a feeling we should be more "gentle" about this.

Reply:
@jmazzitelli that's fair, i'll do it via https://issues.jboss.org/browse/KIALI-2482